frontends/web/src/routes/account/manage/manage.tsx

```diff
@@ -97,8 +99,27 @@ class ManageAccount extends Component<Props, State> {
                 console.info(`${coinCode} selected`);
                 break;
             case 'choose-name':
-                // TODO: post accountName and coinCode to backend
-                console.info(`add new account for ${coinCode} with the name: ${accountName}`);
+                interface ResponseData {
+                    success: boolean;
+                    errorCode?: 'alreadyExists' | 'limitReached';
+                    errorMessage?: string;
+                }
+
+                apiPost('account-add', {
+                    coinCode,
+                    name: accountName,
+                }).then((data: ResponseData) => {
+                    if (data.success) {
+                        this.setState({ step: 2 });
```

Comment:
Shouldn't it be step + 1 ​`this.setState((state) => ({ step: state.step + 1 }));` ?

Otherwise skiping over select coin doesn't work for the bitcoin-only verison

Reply:
So step 2 is not always the last step?

In any case, I think it would help a lot to get rid of the numeric step and switch to logical steps only. If there is only one coin, start at `choose-name`